units/latest.pxu

```diff
@@ -1,104 +1,20 @@
 id: edgex/latest/install
```

Comment:
change the id `install` to any other name would cause some errors:
```
CRITICAL plainbox.crashes: Executable 'checkbox-provider-tools' invoked with Namespace(command=<plainbox.provider_manager.ValidateCommand object at 0x7f7fc09ea128>, debug_console=False, debug_interrupt=False, deprecated=True, log_level=None, new_validation_core=True, pdb=False, strict=True, trace=[]) has crashed
Traceback (most recent call last):
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/impl/clitools.py", line 560, in dispatch_and_catch_exceptions
    return self.dispatch_command(ns)
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/impl/clitools.py", line 556, in dispatch_command
    return ns.command.invoked(ns)
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/provider_manager.py", line 987, in invoked
    return self.invoked_new(ns)
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/provider_manager.py", line 1090, in invoked_new
    for issue in itertools.chain(early_issue_gen, issue_gen):
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/provider_manager.py", line 1139, in validate_units_in_context
    for issue in unit.check(context=context, live=True):
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/impl/unit/unit.py", line 926, in _check_gen
    for issue in validator.check_in_context(self, context):
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/impl/unit/unit.py", line 143, in check_in_context
    self, unit, field, context):
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/impl/unit/validators.py", line 583, in check_in_context
    units_with_this_value = value_map[value]
KeyError: 'com.canonical.certification::edgex/latest/edgexfoundry'
Traceback (most recent call last):
  File "/snap/checkbox-provider-tools/349/bin/checkbox-provider-tools", line 33, in <module>
    sys.exit(load_entry_point('checkbox-ng==1.13.0.dev0', 'console_scripts', 'checkbox-provider-tools')())
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/checkbox_ng/launcher/provider_tools.py", line 30, in main
    spec.loader.exec_module(foo)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/root/parts/checkbox-provider-edgex/build/manage.py", line 13, in <module>
    gettext_domain='checkbox-provider-edgex',
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/provider_manager.py", line 1682, in setup
    raise SystemExit(ProviderManagerTool(definition, kwargs).main())
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/impl/clitools.py", line 324, in main
    return self.dispatch_and_catch_exceptions(ns)
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/impl/clitools.py", line 560, in dispatch_and_catch_exceptions
    return self.dispatch_command(ns)
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/impl/clitools.py", line 556, in dispatch_command
    return ns.command.invoked(ns)
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/provider_manager.py", line 987, in invoked
    return self.invoked_new(ns)
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/provider_manager.py", line 1090, in invoked_new
    for issue in itertools.chain(early_issue_gen, issue_gen):
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/provider_manager.py", line 1139, in validate_units_in_context
    for issue in unit.check(context=context, live=True):
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/impl/unit/unit.py", line 926, in _check_gen
    for issue in validator.check_in_context(self, context):
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/impl/unit/unit.py", line 143, in check_in_context
    self, unit, field, context):
  File "/snap/checkbox-provider-tools/349/lib/python3.6/site-packages/plainbox/impl/unit/validators.py", line 583, in check_in_context
    units_with_this_value = value_map[value]
KeyError: 'com.canonical.certification::edgex/latest/edgexfoundry'
Failed to run 'override-build': Exit code was 1.
```

Reply:
Does it work with checkbox-cli?
https://github.com/canonical/edgex-checkbox-provider#run-tests-using-checkbox-cli

Reply:
I could build without errors. Try cleaning the affected part.

Reply:
I can't run this with checkbox-cli:
```
edgex-checkbox-provider$ sudo ./manage.py install
checkbox-cli: command not found
```
Did you run checkbox-cli without an error?

Reply:
No, it isn't compatible with Python 3.10 and I didn't bother trying others because I could build the snap.

Reply:
> Did you run checkbox-cli without an error?

No, I have Python 3.10 and the script isn't compatible: `AttributeError: module 'collections' has no attribute 'Sequence'`

Reply:
yphus pointed out that this can be solved by removing [this line](https://github.com/canonical/edgex-checkbox-provider/blob/af6646c1aac94fe0865ef0ad6c37c86e6e71e043/snap/snapcraft.yaml#L81) from snapcraft.yaml, which is using an [old copy of the tests](https://github.com/canonical/checkbox/blob/8cdd92cb7b02b64d76f4fe474569abb3f0d6f561/checkbox-core-snap/series20/snap/snapcraft.yaml#L397).

I tried this an it resolved the build error.